src/pages/Facility.tsx

```diff
@@ -30,6 +31,11 @@ export const Facility = () => {
     return createSearchParams(params);
   }, [latestQueryParams, createSearchParams]);
 
+  const FacilityLayout = styled(Box)(({ theme }) => ({
+    paddingLeft: theme.spacing(8),
+    paddingRight: theme.spacing(8)
+  }));
```

Comment:
I would use container to wrap, on small screens padding: 8 could break view, please use implementation as on home page <Container maxWidth='lg'>. Fyi padding should be passed as <MyElement p={8}> . px,py, pt,pb, p and etc

Reply:
I added the padding during a call with Eli yesterday as we tweaked the designs on the spot per her recommendations.
She wanted this on all pages, but I didn't want to break anything at the moment and felt this should be 
discussed with the team first, so I applied to this page only.

Also `px` does not work in `styled`. I tried it